clang/test/CIR/CodeGen/X86/builtin_mask_compress.c

```diff
@@ -0,0 +1,7 @@
+#include <immintrin.h>
+///home/cs25resch11005/myFiles/off_contrib/llvm-project/clang/lib/Headers/avx512vlvbmi2intrin.h
+
+__m128i test_mm_mask_compress(__m128i __S, __mmask8 __U, __m128i __D){
+
+    return (__m128i)_mm_mask_compress_epi16(__S, __U, __D);
```

Comment:
Please add assertions here. You can look at other tests for different builtins. Generally we do three checks:
1. Check that the correct clang IR assembly is generated (CIR checks)
2. Check that the correct LLVM IR is generated after lowering from CIR (LLVM checks)
3. Verify the LLVM IR generated with classing code gen (OGCG checks). This alerts us if something in classic codegen changes, since the tests will fail